src/temperature/crud.py

```diff
@@ -0,0 +1,88 @@
+import os
+from typing import Optional
+from fastapi import HTTPException
+
+from sqlalchemy.ext.asyncio import AsyncSession
+
+from datetime import datetime
+from sqlalchemy.future import select
+import httpx
+from src.temperature.models import Temperature
+from src.city.models import City
+
+from dotenv import load_dotenv
+
+load_dotenv()
+
+API_KEY = os.getenv("WEATHER_API_KEY")
+URL = os.getenv("BASE_URL")
```

Comment:
Ensure that the `WEATHER_API_KEY` and `BASE_URL` environment variables are set correctly. Consider adding error handling to check if these variables are `None` and raise an appropriate exception if they are not set.